### PR TITLE
44 no defaults

### LINKED ISSUE #44: Insure that defaults don't show up on front-end
Make sure all defaults such as "Select a ministry" or "Select a Status" don't show up as real filled out fields on the front-end.

### PR DESCRIPTION
**Pull request by kamicut**
@dereklieu This fixes #44 and separates English and Arabic form values. Can you take a quick look?

### CODE REVIEW

File: app/assets/scripts/components/Project.js [hunk 1]
```diff
@@ -65,6 +65,9 @@ class Project extends React.Component {
         return key !== 'name';
       }).map(function (key) {
         if (!keys[key]) { return <div></div>; }
+        if (keys[key].type === 'object' && 'ar' in keys[key].properties) {
```
Comment (dereklieu) [inline app/assets/scripts/components/Project.js:68]: Is `keys[key]` a custom object structure that has a `type` property?

File: app/assets/scripts/components/ProjectForm.js [hunk 2]
```diff
@@ -498,7 +472,129 @@ class ProjectForm extends React.Component {
...
+            ' المحور الثالث: المعرفة والابتكار والبحث العلمي',
+            ' المحور الرابع: شفافية وكفاءة المؤسسات الحكومية',
+            ' المحور الخامس: العدالة الاجتماعية',
+            ' المحور السادس: الصحة',
+            ' المحور السابع: التعليم والتدريب',
+            ' المحور الثامن: الثقافة',
+            ' المحور التاسع: البيئة',
+            ' المحور العاشر: التنمية العمرانية',
+            '',
+            ''
```
Comment (dereklieu) [inline app/assets/scripts/components/ProjectForm.js:523]: Do these last two empty string items need to be here? Also it seems like each arabic string has leading whitespace, is that right?

**Comment by dereklieu** [comment]
Left two minor questions inline. Guessing the empty spaces are because we don't have arabic translation strings for everything? 

Besides that nothing jumped out on me, but I didn't have time to do a manual test.

File: app/assets/scripts/components/Project.js (hunk 1 above)
Comment (kamicut) [reply app/assets/scripts/components/Project.js:68]: keys[key] refers to an entry in the schema properties [here](https://github.com/map-egypt/admin/blob/master/app/assets/scripts/components/ProjectForm.js#L8)

File: app/assets/scripts/components/ProjectForm.js (hunk 2 above)
Comment (kamicut) [reply app/assets/scripts/components/ProjectForm.js:523]: I removed the whitespace. The empty strings have to be there to match the number of items in each dropdown. They don't have equivalent translation strings.

**Pull request merged**